europe/README.EUROPE

```diff
@@ -0,0 +1,103 @@
+*Format*: GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Transform ellipsoidal heights to the local height system DVR90. The horizontal
+grid coordinates are referenced to ETRS89.
+
+* dvr90.gtx
+
+### Faroe Islands: GRS80 ellipsoidal heights -> FVR09
+
+*Source*: [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
+*Format*: GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Transform ellipsoidal heights to the local height system FVR09. The horizontal
+grid coordinates are referenced to ETRS89.
+
+* fvr09.gtx
+
+### Nordic+Baltic countries: Deformation model
+
+*Source*: [The Nordic Geodetic Commision](https://github.com/NordicGeodesy/NordicTransformations)  
+*Format*: CTable2 and GTX  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Deformation model covering the Nordic and Baltic countries. Used in
+transformations between global reference frames and the local realisations
+of ETRS89 in the Nordic and Baltic countries. See the `NKG` init-file for
+examples of use.
+
+* nkgrf03vel_realigned_xy.ct2
+* nkgrf03vel_realigned_z.gtx
+
+## Included init-files
+
+### Denmark: System definitions related to ETRS89
+
+*Source*: [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Init files with various transformations and conversions of relevance in
+Denmark. All definitions in the init file uses the local ETRS89 reference
+frame as a pivot datum. More information can be found in the
+[NordicTransformations](https://github.com/NordicGeodesy/NordicTransformations)
+repository. While the `.pol` files are init-files they are not supposed to be
+by themselves. The exist purely as helper files for the `DK` init-file.
+The `.pol` files holds polynomial constants used for transformations between
+Danish legacy coordinate reference systems and ETRS89.
+
+* DK
+* DK_bornholm.pol
+* DK_bridges.pol
+* DK_general.pol
+* DK_jutland.pol
+* DK_zealand.pol
+
+### The Faroe Islands: System definitions related to ETRS89
+
+*Source*: [Agency for Data Supply and Efficiency](https://github.com/NordicGeodesy/NordicTransformations)  
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Init file with various transformations and conversions of relevance in
+The Faroe Islands. All definitions in the init file uses the local ETRS89
+reference frame as a pivot datum. More information can be found in the
+[NordicTransformations](https://github.com/NordicGeodesy/NordicTransformations)
+repository.
+
+* FO
+
+### Nordic+Baltic countries: ITRFxx -> Local ETRS89 realizations
+
+*Source*: [The Nordic Geodetic Commision](https://github.com/NordicGeodesy/NordicTransformations)  
```

Comment:
s/Commision/Commission/, unless the typo is part of the offical name.

Reply:
Nah, luckily it's just my typo :)